notebooks/03-query/21-query-async-historical.ipynb

```diff
@@ -0,0 +1,610 @@
+    "    \"type\": \"loadByInterval\",\n",
+    "    \"interval\": \"2005-11-18/P10Y\",\n",
+    "    \"tieredReplicants\": {\n",
+    "      \"_default_tier\": 1\n",
+    "    }\n",
+    "  },\n",
+    "  {\n",
+    "    \"type\": \"loadForever\",\n",
+    "    \"tieredReplicants\": {},\n",
+    "    \"useDefaultTierForNull\": \"false\"\n",
+    "  }\n",
+    "]\n",
+    "\n",
+    "requests.post(f\"{druid_host}/druid/coordinator/v1/rules/{table_name}\", json.dumps(retention_rules), headers=druid_headers)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20db806c-abca-41cd-ab9e-0b88d655633f",
+   "metadata": {},
+   "source": [
+    "Now run the next cell to confirm the change."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f34f80b4-601c-4a27-b0c2-6c495a262105",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql=f'''\n",
+    "SELECT\n",
+    "  a.\"start\",\n",
+    "  a.\"end\",\n",
+    "  c.\"server\",\n",
+    "  c.\"tier\",\n",
+    "  \"num_rows\",\n",
+    "  \"size\"\n",
+    "FROM \"sys\".\"segments\" a\n",
+    "LEFT JOIN \"sys\".\"server_segments\" b ON a.\"segment_id\" = b.\"segment_id\"\n",
+    "LEFT JOIN \"sys\".\"servers\" c ON b.\"server\" = c.\"server\"\n",
+    "WHERE \"datasource\" = '{table_name}'\n",
+    "ORDER BY \"start\", \"tier\"\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d88e9033-f322-4c62-ab05-78db3dea5c45",
+   "metadata": {},
+   "source": [
+    "Run the cell above until some segments are shown without a server and tier.\n",
+    "\n",
+    "Notice that, when you run the following query, the results exclusively consist of data that has been pre-fetched to a historical."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "af4affc5-bfcd-49fb-84a7-40bb810b2b09",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql=f'''\n",
+    "SELECT\n",
+    "  TIME_FLOOR(\"__time\",'P1D') AS \"period\",\n",
+    "  COUNT(*) as \"events\"\n",
+    "FROM \"{table_name}\"\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b6c9b88-837d-4c80-a28d-36184ba63355",
+   "metadata": {},
+   "source": [
+    "## Execute an asynchronous query\n",
+    "\n",
+    "Use the `/druid/v2/sql/statements` API endpoint to run asynchronous queries using MSQ engine."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "045df49c-24ad-4eff-9b5f-0a30bfd47bf1",
+   "metadata": {},
+   "source": [
+    "### Call the API using the druid_api package\n",
+    "\n",
+    "The `async_sql` method handles the [necessary steps](https://druid.apache.org/docs/latest/tutorials/tutorial-query-deep-storage#query-from-deep-storage) not only to submit the query, but to retrieve the results.\n",
+    "\n",
+    "Run the following cell, which uses the `async_sql` method of the `druid_api` package to call the API and return the results."
```

Comment:
```suggestion
    "Run the following cell to run an ansynchronous query and return the results."
```
I think line 330 is clear enough about the `async_sql` method so we don't need to repeat it here.